schema/types/Life.graphql

```diff
@@ -0,0 +1,28 @@
+type Life {
+    id: ObjectID!
+    firstName: String!
+    lastName: String!
+    fullName: String!
+    title: String!
+    description: String!
+    birthday: DateTime!
+    hobbies: [String!]!
+}
+
+input LifeInput {
+    firstName: String!
+    lastName: String!
+    description: String!
+    title: String!
+    birthday: DateTime!
+    hobbies: [String!]!
+}
+
+input EditLifeInput {
+    firstName: String
+    lastName: String
+    description: String
+    title: String
+    birthday: DateTime
+    hobbies: [String]
+}
```

Comment:
you can instead reuse the `input` if they have the same fields

Reply:
I created a separate input for edit life in light of the case where no fields were entered to edit - in essence allowing GraphQL to accept null arguments when editing. Would this method work?